### DISCUSSION TITLE
Welcome to redis-codesys-lib Discussions!

### DISCUSSION THREAD

**Discussion by thorgrimjansrud** [thorgrimjansrud/redis-codesys-lib#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GiuseppeGambacorta** [comment]
Hi!
Why did you share only the compiled library and not the CODESYS project? :)

**Reply by thorgrimjansrud** [reply]
Hi, the main reason is that if the source code is open-sourced it's because I want others to contribute (not just have the code). This would be difficult to achieve/maintain by a codesys project, or PLCopen XML, because of the format. The plan is to wait for the CODESYS go plattform. Hopefully we get some of it this year. Anyway, hope you like it!

Codesys
https://www.codesys.com › 1...PDF
CODESYS go!
